a template argument list is expected after a name prefixed by the template keyword

diff -urN VTK-9.3.1/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/internal/VariantImplDetail.h.orig VTK-9.3.1/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/internal/VariantImplDetail.h
--- VTK-9.3.1/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/internal/VariantImplDetail.h.orig	2024-06-28 18:00:10.000000000 +0200
+++ VTK-9.3.1/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/internal/VariantImplDetail.h	2025-01-13 15:35:29.812473900 +0100
@@ -548,7 +548,7 @@
         // just the one that it currently holds.
         return f(storage.V7, std::forward<Args>(args)...);
       default:
-        return VariantCases<NumCases - 8>::template CastAndCall(
+        return VariantCases<NumCases - 8>::template CastAndCall<>(
           index - 8, std::forward<Functor>(f), storage.Remaining, std::forward<Args>(args)...);
     }
   }
